v3/src/components/graph/v2-graph-exporter.ts

```diff
@@ -230,13 +231,34 @@ function getPlotModels(graph: IGraphContentModel): Partial<ICodapV2GraphStorage>
   return storage
 }
 
+// v2 uses color names for default stroke colors
+const strokeColorStr = (color: string) => {
+  const colorHex = removeTransparencyFromColor(color).toLowerCase()
+  return colorHex === "#ffffff" ? "white"
+            : colorHex === "#d3d3d3" ? "lightgrey"
+            : colorHex
+}
+
 export const v2GraphExporter: V2TileExportFn = ({ tile }) => {
   const graph = isGraphContentModel(tile.content) ? tile.content : undefined
   if (!graph) return
 
   const componentStorage: Partial<ICodapV2GraphStorage> = {
     _links_: getLinks(graph),
     displayOnlySelected: !!graph.dataConfiguration.displayOnlySelectedCases,
+    pointColor: removeTransparencyFromColor(graph.pointDescription.pointColor),
+    transparency: getTransparency(graph.pointDescription.pointColor, "point"),
+    strokeColor: graph.pointDescription.pointStrokeSameAsFill
+                    ? "white" // v2 uses white for stroke when stroke is same as fill
+                    : strokeColorStr(graph.pointDescription.pointStrokeColor),
+    strokeTransparency: graph.pointDescription.pointStrokeSameAsFill
+                          ? 0.4 : getTransparency(graph.pointDescription.pointStrokeColor, "stroke"),
+    pointSizeMultiplier: graph.pointDescription.pointSizeMultiplier,
+    strokeSameAsFill: graph.pointDescription.pointStrokeSameAsFill,
+    plotBackgroundColor: graph.plotBackgroundColor === "#FFFFFF"
```

Comment:
Do we need to worry about other variations (e.g. `#ffffff` or `white`?) or is this string canonicalized to this format?